tests/sentry/issues/auto_source_code_config/test_process_event.py

```diff
@@ -81,7 +81,8 @@ def _process_and_assert_code_mapping(
         expected_source_root: str,
         expected_repo_name: str = REPO1,
         expected_num_code_mappings: int = 1,
-    ) -> None:
+        expected_in_app_stack_trace_rules: list[str] | None = None,
+    ) -> GroupEvent:
```

Comment:
We return the event to inspect information about the grouping hashes (in the following PR).